shared/login/forms/container.desktop.js

```diff
@@ -34,6 +34,7 @@ const styles = {
     height: '100%'
   },
   button: {
-    zIndex: 9999
+    zIndex: 9999,
+    position: 'absolute'
```

Comment:
Mmm be careful here about the layering. I think we had this before but it wasn't clickable. Just double check